README.md

```diff
@@ -14,7 +15,42 @@ to automatically build an Ubuntu 18.04-based 'sancus-devel' container. Simply
 execute `make docker` to build and run the Docker container, or see the
 [docker](docker) subdirectory for detailed instructions.
 
-## Requirements and Dependencies:
+## Quickstart
+
+If you do not care about modifying the Sancus code itself but just wish to work with the existing toolchain, using the Docker images may already be enough for you!
+
+```bash
+# Pull latest Sancus image for 128 bit security
+$ docker pull ghcr.io/fritzalder/sancus-main/sancus-devel-128:latest
+# Run Docker interactively
+$ docker run -it ghcr.io/fritzalder/sancus-main/sancus-devel-128:latest
+# Run Docker and attach directory ~/project into the Docker file system
+$ docker run -it -v ~/project:/sancus/project ghcr.io/fritzalder/sancus-main/sancus-devel-128:latest
+# Run Docker and attach above directory but also forward USB UART0 to the container
+$ docker run -it -v ~/project:/sancus/project --device /dev/ttyUSB0 ghcr.io/fritzalder/sancus-main/sancus-devel-128:latest
```

Comment:
shouldn't the image name be `ghcr.io/sancus-tee/sancus-main/sancus-devel-128` ?

Reply:
Oh damn good catch. I somehow must have had my repo open when copying that line from GitHub. I'm not trying to run a supply chain attack here,I promise! :D

Reply:
Fixed it, thanks